dbt-redshift/tests/functional/adapter/test_exception_handling.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from dbt.tests.util import run_dbt_and_capture
+
+_TEST_MODEL = """
+                {{
+                  config(
+                    materialized = 'table',
+                    )
+                }}
+                select 1 as test_col
+            """
+
+_MACROS = {
```

Comment:
Now that we're re-running tests, can we make this format follow the models format? i.e. create the macro itself as the constant and assign it:

```python
_TEST_MACRO = """
{% macro transaction_macro() %}
    ...
{% endmacro %}
```
```python
    @pytest.fixture(scope="class")
    def macros(self):
        return {"macro.sql": _TEST_MACRO}
```